Resources/Prototypes/_CM14/Entities/Structures/Piping/atmospherics.yml

```diff
@@ -0,0 +1,121 @@
+- type: entity
+  parent: GasUnaryBase
+  id: CMVentPump
+  name: vent
+  description: Has a valve and pump attached to it.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    drawdepth: FloorObjects
+    sprite: _CM14/Structures/Piping/Atmospherics/vent_pump.rsi
+    layers:
+        - sprite: Structures/Piping/Atmospherics/pipe.rsi
+          state: pipeHalf
+          map: [ "enum.PipeVisualLayers.Pipe" ]
+        - state: off
+          map: [ "enabled", "enum.SubfloorLayers.FirstLayer" ]
```

Comment:
These layers are indenting to outer space
If you are using Rider you can press Ctrl+Alt+L to reformat the whole file